components/faction/commons/faction.lua

```diff
@@ -51,17 +53,49 @@ end
 
 --- Parses a faction from input. Returns the factions short handle/identifier.
 -- Returns nil if not a valid faction.
+-- If `options.alias` is set to false the function will not look in the aliases provided via the data module.
 ---@param faction string
+---@param options {alias: boolean?}?
 ---@return string?
-function Faction.read(faction)
+function Faction.read(faction, options)
 	if type(faction) ~= 'string' then
 		return nil
 	end
 
+	options = options or {}
+
 	faction = faction:lower()
 	return Faction.isValid(faction) and faction
 		or byLowerName[faction]
-		or Faction.aliases[faction]
+		or (options.alias ~= false and Faction.aliases[faction])
+		or nil
+end
+
+--- Parses multiple factions from input.
+-- Returns a table of the found factions short handle/identifier.
+-- Returns empty table if no input is specified.
+---@param input: string
+---@param options: {sep: string?, alias: boolean?}?
+---@return table
+function Faction.readMultiFaction(input, options)
+	if String.isEmpty(input) then
+		return {}
+	end
+
+	options = options or {}
+
+	if Faction.read(input, options) then
+		return {Faction.read(input, options)}
+	end
```

Comment:
```suggestion
	local faction = Faction.read(input, options)
	if faction then return faction end
```

Not that it really matters tho. Just a bit weird seeing two calls to the same thing

Reply:
linter: shadowing faction
will rename var

Reply:
I disable variable shadow warnings in my other projects, since it comes up frequently and are almost always false positives.